numalogic/udfs/tools.py

```diff
@@ -88,7 +110,7 @@ def make_stream_payload(
 # TODO: move to base NumalogicUDF class and look into payload mutation
 def _get_artifact_stats(artifact_data):
     return {
-        "source": None or artifact_data.extras.get("source"),
+        "artifact_source": None or artifact_data.extras.get("source"),
```

Comment:
Why is `None` checked before the real data?

Reply:
fixed!